app/src/main/java/de/danoeh/antennapod/fragment/EpisodesListFragment.java

```diff
@@ -214,6 +217,12 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
             ((SimpleItemAnimator) animator).setSupportsChangeAnimations(false);
         }
 
+        SwipeRefreshLayout swipeRefreshLayout = root.findViewById(R.id.swipeRefresh);
+        swipeRefreshLayout.setOnRefreshListener(() -> {
+            AutoUpdateManager.runImmediate(requireContext());
+            new Handler().postDelayed(() -> swipeRefreshLayout.setRefreshing(false), 1000);
```

Comment:
1 second feels a bit too much to me. This is sometimes more than what it takes to actually refresh. What about 500ms?